ballerina/annotation_processor.bal

```diff
@@ -104,6 +104,13 @@ isolated function getFieldCacheConfig(service object {} serviceObj, parser:RootO
     return;
 }
 
+isolated function getDocumentCacheConfig(GraphqlServiceConfig? serviceConfig) returns DocumentCacheConfig? {
```

Comment:
Do we need this function? I think we are directly accessing this for the other fields? (I may have mistaken though, maybe that change is in the master branch).